vnext/local-cli/generator-windows/templates/cpp/src/MainPage.xaml

```diff
@@ -0,0 +1,15 @@
+﻿<Page
+    x:Class="<%=ns%>.MainPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:<%=ns%>"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+      xmlns:react="using:react.uwp"
+    mc:Ignorable="d"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+
+    <Grid>
```

Comment:
>   [](start = 2, length = 2)

nit: remove two spaces

Reply:
fixed

---
In reply to: [329279850](https://github.com/microsoft/react-native-windows/pull/3262#discussion_r329279850) [](ancestors = 329279850)